fakeserver/server.go

```diff
@@ -64,7 +57,23 @@ func Start(listenOn string) {
 			}
 			return false
 		},
-	}).Handler(r)))
+	}).Handler(next)
+}
+
+// Start the server
+func Start(listenOn string, logRequests bool) {
+	r := mux.NewRouter()
+
+	s := &server{router: r}
+	s.routes()
+
+	if logRequests {
+		r.Use(loggingMiddleware)
+	}
+	r.Use(corsMiddleware)
```

Comment:
refactored to use mux's `Use` functionality which allows you to apply middleware handlers